.gitignore

```diff
@@ -18,6 +18,7 @@
 /build
 
 # misc
+.vscode
```

Comment:
I think you need to add a `/` in front of this, so that it knows it's a folder, not a file.

Reply:
alright making the change and pushing momentarily